src/Views/Blog/Post.cshtml

```diff
@@ -32,7 +32,7 @@
                 <li> Posted in </li>
                 @foreach (string cat in Model.Categories)
                 {
-                    <li itemprop="articleSection"><a asp-controller="Blog" asp-action="Category" asp-route-category="@cat.ToLowerInvariant()">@cat</a></li>
```

Comment:
This makes the url mixed case, right? Not lowecase as it should be

Reply:
the url is build now: /blog/category/@cat.ToLowerInvariant()/
before is was: /blog/category/@cat.ToLowerInvariant()/page_from_current_context 
>this page is wrong here<